modules/sdk-coin-xlm/src/lib/utils.ts

```diff
@@ -1,22 +1,48 @@
 import { isValidEd25519PublicKey, isValidEd25519SecretKey } from '@bitgo/sdk-core';
 import * as stellar from 'stellar-sdk';
 
+/**
+ * Creates a Stellar keypair from a public key.
+ * Public key can be either a raw root public key (hex string) or a Stellar public key (prefixed with 'G')
+ * @param pub Either hex root public key or Stellar public key
+ * @returns Stellar keypair from the provided pub
+ */
+export function createStellarKeypairFromPub(pub: string): stellar.Keypair {
+  if (pub.startsWith('G')) {
+    return stellar.Keypair.fromPublicKey(pub);
+  }
+  return stellar.Keypair.fromPublicKey(encodePublicKey(Buffer.from(pub, 'hex')));
+}
+
+/**
+ * Creates a Stellar keypair from a private key.
+ * Private key can be either a raw root private key (hex string) or a Stellar private key (prefixed with 'S').
+ * @param secret Either hex root private key or Stellar private key
+ * @returns Stellar keypair from the provided prv
+ */
+export function createStellarKeypairFromPrv(prv: string): stellar.Keypair {
+  if (prv.startsWith('S')) {
+    return stellar.Keypair.fromSecret(prv);
+  }
+  return stellar.Keypair.fromSecret(encodePrivateKey(Buffer.from(prv.slice(0, 64), 'hex')));
+}
```

Comment:
Lets add tests for these utils.

Reply:
There are some tests for these, they are in the `utils.ts` test file at the bottom